app/templates/road-sign-concepts/road-sign-concept.hbs

```diff
@@ -97,10 +97,28 @@
             {{/if}}
           </dd>
         </div>
+
+        <div>
+          <dt class="au-u-h5 au-u-medium">
+            {{t "utility.variables"}}
+          </dt>
+          <dd class="au-u-muted">
+            {{#if @model.roadSignConcept.variables}}
+              <ul>
+                {{#each @model.roadSignConcept.variables as |variable|}}
+                  <li>
+                    {{variable.value}}
+                  </li>
+                {{/each}}
+              </ul>
```

Comment:
I would add an `{{else}}` here, similar to the shapes. Or alternatively hide the whole <div> so the title isn't shown either.

Reply:
Yes noticed that yesterday aswell so added a '-' when it is empty like on the shape manager